### PR TITLE
CI Travis: Check coding style with clang-format

### PR DESCRIPTION
**Pull request by krocard**
Check that the coding style is respected with the CI.
Note that coding style infringent sources will still be tested but the build will be marked as failed.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/325%23issuecomment-169019500%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/325%23issuecomment-169031668%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/325%23issuecomment-169297490%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/325%23issuecomment-169326136%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/325%23discussion_r48850329%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/325%23discussion_r48854189%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/325%23issuecomment-169019500%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A-1%3A%20%22%2C%20%22created_at%22%3A%20%222016-01-05T14%3A42%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%40dawagner%20Please%20review%20again.%22%2C%20%22created_at%22%3A%20%222016-01-05T15%3A15%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40dawagner%20please%20review%22%2C%20%22created_at%22%3A%20%222016-01-06T10%3A59%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222016-01-06T13%3A45%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%208a44a1a734149ef49d3946055b169758465f74b8%20.travis.yml%2021%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/325%23discussion_r48850329%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22We%20have%20a%20lot%20of%20%5C%22.h%5C%22%20files%20that%20are%20C%2B%2B%20headers.%22%2C%20%22created_at%22%3A%20%222016-01-05T14%3A40%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22changed%20to%20%60%5C%5C.%5Bch%5D%28pp%29%3F%24%60%20%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222016-01-05T15%3A14%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20.travis.yml%3AL55-65%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/dawagner%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/dawagner'></a>

- [x] <a href='#crh-comment-Pull 8a44a1a734149ef49d3946055b169758465f74b8 .travis.yml 21'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/325#discussion_r48850329'>File: .travis.yml:L55-65</a></b>
- <a href='https://github.com/dawagner'></a> We have a lot of ".h" files that are C++ headers.
- <a href='https://github.com/krocard'></a> changed to `\.[ch](pp)?$` :+1:
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/325#issuecomment-169019500'>General Comment</a></b>
- <a href='https://github.com/dawagner'></a> :-1:
- <a href='https://github.com/krocard'></a> :+1: @dawagner Please review again.
- <a href='https://github.com/krocard'></a> @dawagner please review

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/325?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/325?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/325?approve=0'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/325'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/325'>Refresh</a>.<a href='#crh-update-none'></a>

File: .travis.yml [hunk 1]
```diff
@@ -53,6 +55,10 @@ before_script:
 
 # how to build
 script:
+  # Check coding style
+  - git ls-files | grep -E '\.*[ch]pp' |  xargs clang-format-3.8 -i &&
```
Comment (dawagner) [inline .travis.yml]: We have a lot of ".h" files that are C++ headers.

**Comment by dawagner** [comment]
:-1:

File: .travis.yml (hunk 1 above)
Comment (krocard) [inline .travis.yml]: changed to `\.[ch](pp)?$` :+1:

**Comment by krocard** [comment]
:+1: @dawagner Please review again.

**Comment by krocard** [comment]
@dawagner please review

**Comment by dawagner** [comment]
:shipit:

**Pull request merged**